src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/ProviderExtensions.cs

```diff
@@ -13,19 +13,19 @@ public static void AddWellcomeProvider(this ResourceBase iiifResource, string ho
         {
             var agent = new Agent
             {
-                Id = "https://wellcomecollection.org",
+                Id = Constants.WellcomeCollectionUri,
                 Label = Lang.Map("en",
-                    "Wellcome Collection",
+                    Constants.WellcomeCollection,
                     "183 Euston Road",
                     "London NW1 2BE UK",
                     "T +44(0)2076118722",
                     "E library@wellcome.ac.uk",
```

Comment:
This should be library@wellcomecollection.org
(from https://wellcomelibrary.org/using-the-library/services-and-facilities/contact-us/)
and spaces in the telephone number 
+44 (0)20 7611 8722

Reply:
Updated contact details.